src/dataset/movie_rating.py

```
@@ -0,0 +1,189 @@
+import re
+from collections import defaultdict
+import json
+
+sys.path.append(
+    os.path.dirname(
+        os.path.dirname(
+            os.path.dirname(os.path.abspath(__file__))))
+)
+
+import pandas as pd
+import numpy as np
```

Ruff (0.12.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)